monolith/views/stories.py

```diff
@@ -176,7 +176,10 @@ def new_stories():
 @login_required
 def write_story(story_id):
     story = Story.query.get(story_id)
-
+    if story is None:
+        abort(404)
```

Comment:
Better to substitute with` message = 'Story not found'`

Reply:
ok! For simplicity I'll add a redirect to the user's wall with this message, in order to have a render template to work on (otherwise the message would have been displayed with all the stuff inside the "editing stories page" --> /write_story/<story_id>)
It's a good idea?

Reply:
Till now we displayed just this message. I think we should follow the line that we take till now and then, if needed, modify all.

Reply:
if that story is null, you should add:
```
message = 'Story not found'
return render_template("message.html", message=message)
```